test/CMakeLists.txt

```diff
@@ -1756,7 +1756,8 @@ COMMAND ${IMPLICITGEMM_TESTING_ENV} MIOPEN_LOG_LEVEL=5 $<TARGET_FILE:test_conv2d
 
 set(CONV_CK_IGEMM_FWD_V6R1_DLOPS_NCHW_ENV
```

Comment:
[Recommendation] Can you please add comment from https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2306#issuecomment-1668655231 to explain why we use env var to enable the solver.

Reply:
Done. Next time I will do them all at once instead of separate commits ;-P